lib/components/form/batch-styled.ts

```diff
@@ -73,10 +82,31 @@ export const PlanTripButton = styled(Button)`
   }
 `
 
+export const ModeSelectorContainer = styled.div<{ squashed?: boolean }>`
```

Comment:
Nice-to-have: The top margin of the `<input>` elements, set by bootstrap, should probably be removed in the popups.

Reply:
That makes the alignment much better! Thank you!